projects/plugins/social/src/class-jetpack-social.php

```diff
@@ -256,16 +257,32 @@ public function initial_state() {
 					'connectionData'  => array(
 						'connections' => $publicize->get_all_connections_for_user(), // TODO: Sanitize the array
 						'adminUrl'    => esc_url_raw( $publicize->publicize_connections_url( 'jetpack-social-connections-admin-page' ) ),
+						'services'    => $this->get_services(),
```

Comment:
Since the method is already defined in settings class. We can use this and get rid of `get_services` in this class.
```suggestion
						'services'    => $jetpack_social_settings->get_services(),
```